### PR TITLE
Insertion test

### LINKED ISSUE #121: Create insertion test
Create a test with insertion of large number of sample, e.g. like in [atsd-api-test](https://github.com/axibase/atsd-api-test/blob/master/src/test/java/com/axibase/tsd/api/method/series/SeriesQueryTest.java#L211). It probably should fail. To enforce the order, use OrderedDict or list of tuples during serialization instead of default dictionary

### PR DESCRIPTION
**Pull request by Arthasasyan**
Resolves #121 
Related to 6566 in Redmine

### CODE REVIEW

**Comment by raipc** [comment]
Ok. It seems like the limitation on fields order was removed in r. 24030. Please, modify the test to insert large dataset using SeriesQuery object. Also, I don't think date needs to be global.

**Comment by Arthasasyan** [comment]
I suppose, it should be not `SeriesQuery`, but `Series` object to insert
As we use default insert method, creating another TestCase is not necessary, will delete it

**Comment by raipc** [comment]
> I suppose, it should be not `SeriesQuery`, but `Series` object to insert
> As we use default insert method, creating another TestCase is not necessary, will delete it

Yes, you are right

**Comment by Arthasasyan** [comment]
Last commit name is senseless, forgot to change it from default, my apologies

File: tests/test_series_service.py [hunk 1]
```diff
@@ -203,3 +203,12 @@ def test_sample_filter(self):
...
         self.assertEqual(2, s.get_last_value())
+
+    def test_large_insert(self):
+        date = datetime.now()
+        data = []
+        for i in range(10 ** 6):
+            data.append(Sample(1, date))
+            date = date - timedelta(minutes=1)
+        series_to_insert = Series(ENTITY, METRIC, data)
+        self.service.insert(series_to_insert)
```
Comment (raipc) [inline tests/test_series_service.py]: Add asertion

Comment (Arthasasyan) [reply tests/test_series_service.py]: In the current realization assertion is not needed - if query is not accepted, an exception will be raised and test will be failed

Comment (raipc) [reply tests/test_series_service.py]: So, the assertions must be "query executes without error" and "insert method returns a correct value"

**Pull request merged**